server/src/main/java/com/hedera/block/server/pbj/PbjBlockStreamServiceProxy.java

```diff
@@ -97,15 +99,15 @@ public Pipeline<? super Bytes> open(
             return switch (m) {
                 case publishBlockStream -> {
                     notifier.unsubscribeAllExpired();
-                    yield Pipelines.<PublishStreamRequest, PublishStreamResponse>bidiStreaming()
+                    yield Pipelines.<List<BlockItemUnparsed>, PublishStreamResponse>bidiStreaming()
```

Comment:
Why change from `PublishStreamRequrest` to `List<BlockItemUnparsed>`?

Reply:
Our main code in `ProducerBlockItemObserver` is only concerned with a list of inbound items. With the `unparsed.proto` approach, you must selectively step down layer by layer to marshal only the objects you need to make decisions. I've centralized those translations here.